src/databricks/labs/ucx/queries/progress/02_1_migration_status_by_owner_bar_graph.sql

```diff
@@ -0,0 +1,22 @@
+/*
+--title 'Tables and views not migrated'
+--overrides '{"spec": {
+    "version": 3,
+    "widgetType": "bar",
+    "encodings": {
+        "x":{"fieldName": "owner", "scale": {"type": "categorical"}, "displayName": "owner"},
+        "y":{"fieldName": "count", "scale": {"type": "quantitative"}, "displayName": "count"}
+    }
+}}'
+*/
+WITH migration_statuses AS (
+    SELECT *
+    FROM inventory.historical
+    WHERE object_type = "migration_status"
+)
+
+SELECT
+    owner,
+    COUNT_IF(SIZE(failures) > 0) AS count
+FROM migration_statuses
+GROUP BY owner
```

Comment:
Do we want to show owners that have a zero count?

If we don't, this can be more efficient by `WHERE size(failures) > 0` and then just using `count(1) as count`.